.rubocop.yml

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
There are some tests failing. Kindly make sure that all the tests are passing.
![fails](https://user-images.githubusercontent.com/35504149/186732799-f06a49ef-136c-4386-ac05-5da992ec1338.png)